src/from-scratch.js

```diff
@@ -1,16 +1,39 @@
-const resolvedWrapper = () => {
+const resolvedWrapper = (value) => {
+  return Promise.resolve(value);
 };
 
-const rejectedWrapper = () => {
+const rejectedWrapper = (errorMessage) => {
+  const error = new Error(errorMessage); // Create a new Error object with the given message
+  return Promise.reject(error);
 };
 
-const handleResolvedPromise = () => {
+const handleResolvedPromise = (value) => {
+  return value.then((value) => {
+    console.log(value);
+    return value.toUpperCase();
+  });
```

Comment:
Rename the parameter from `value` to `promise` for clarity:

```suggestion
const handleResolvedPromise = (promise) => {
  return promise.then((value) => {
    console.log(value);
    return value.toUpperCase();
  });
```

Also, can you add a comment to explain why you have to return from inside the .then callback AND why you're returning `promise.then` from the entire function?